WalletWasabi/WabiSabi/WabiSabiCoordinator.cs

```diff
@@ -26,6 +23,7 @@ public WabiSabiCoordinator(CoordinatorParameters parameters, IRPCClient rpc)
 			Warden = new(parameters.UtxoWardenPeriod, parameters.PrisonFilePath, Config);
 			ConfigWatcher = new(parameters.ConfigChangeMonitoringPeriod, Config, () => Logger.LogInfo("WabiSabi configuration has changed."));
 
+			CoinJoinTransactionArchiver transactionArchiver = new(Path.Combine(parameters.CoordinatorDataDir, "CoinJoinTransactions"));
```

Comment:
`transactionArchiver` is created, but not sent to the `Arena` one line below.
Was that on purpose?

Reply:
nope, tx.